cssdb.json

```diff
@@ -229,15 +212,16 @@
     "specification": "https://www.w3.org/TR/css-color-4/#color-adjust",
```

Comment:
`color-adjust` was moved to separate spec - https://www.w3.org/TR/css-color-adjust-1/

Reply:
Thanks!